cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsResourceConnector.java

```diff
@@ -162,7 +162,7 @@ public Object collectResourcesToRemove(AuthenticatedContext authenticatedContext
     @Override
     public List<CloudResourceStatus> downscale(AuthenticatedContext auth, CloudStack stack, List<CloudResource> resources, List<CloudInstance> vms,
             Object resourcesToRemove) {
-        return awsDownscaleService.downscale(auth, stack, resources, vms, resourcesToRemove);
```

Comment:
The `resourcesToRemove` parameter comes from the interface, and does seem to be used by by other cloud providers, just not AWS. However, I found it very confusing when I was working on this to see that `awsDownscaleService.downscale` had a parameter called `resourcesToRemove`, but if you look at the actual code those aren't the resources being removed because that object isn't used at all. So this technically doesn't have anything to do with my change, but I thought it cleaned up the code and made it less confusing.

Reply:
Ah, ok. You're right, this is better I think.
The real issue is with the interface, but I'm not familiar enough with it to suggest a refactoring that would work.

Thanks!